pkg/scheduler/core/generic_scheduler.go

```diff
@@ -67,7 +72,12 @@ func (g *genericScheduler) Schedule(ctx context.Context, placement *policyv1alph
 	klog.V(4).Infof("feasible clusters scores: %v", clustersScore)
 
 	clusters := g.selectClusters(clustersScore, placement.SpreadConstraints, feasibleClusters)
-	result.SuggestedClusters = clusters
+
+	clustersWithReplicase, err := g.assignReplicas(clusters, placement.ReplicaSchedulingStrategy, object)
```

Comment:
It seems like we need add a condition here, such as "if resourceBinding.Spec.ReplicasBinding > 0". For resource like service, secret can only support duplicate mode.

Reply:
OK, I will add。But I think it will be better that this condition is in the func assignReplicas.